govuk-paas/paas-cdn-broker
# [#177003129] Show configuration details in response to `cf service SERVICE_NAME`

What
---

The `GetInstance` method of the broker spec is called when a user does `cf service SERVICE_NAME`. It allows the broker to describe thing service instance, and the results are shown to the user.

How to review
---
1. Code review
2. Deploy and run (currently deployed to dev02)
    * Check it works for new instances
    * Check it works for existing instances

`cf service SERVICE_NAME` doesn't show the details, unfortunately. Call `cf curl "/v3/service_instances/$(cf service SERVICE_NAME --guid)/parameters"`

Post review
---
Update [the Bosh release](https://github.com/alphagov/paas-cdn-broker-boshrelease/pull/45) to point at the merged commit.

Update [the `paas-cf` PR to use the proper Bosh release version](https://github.com/alphagov/paas-cf/pull/2826)
